src/pages/Stream/providers/LogsProvider.tsx

```diff
@@ -810,6 +811,18 @@ const setAndFilterData = (store: LogsStore, filterKey: string, filterValues: str
 	};
 };
 
+export const updateTimeRange = (store: LogsStore) => {
```

Comment:
Context reducer fn should not be exported like this. 
Add this to module export. And why do we need to call another reducer fn inside this? Give a meaningful name to the fn and handle every logic inside it. 

what if I want to change the logic of `setTimeRange` in the future? Let's keep the dependency on each other fns to minimum.

Reply:
Moved the logic to a new reducer.